install/kubernetes/cilium/templates/cilium-flowlog-configmap.yaml

```diff
@@ -12,5 +12,17 @@ metadata:
 data:
   flowlogs.yaml: |
     flowLogs:
-{{ .Values.hubble.export.dynamic.config.content | toYaml | indent 4 }}
+      {{- /* hubble export configurations moved, use default to make upgrades seemless */ -}}
```

Comment:
Should we add a "TODO remove once v1.18 is released" note here, so this doesn't stick around forever?

Reply:
Yes will do